modules/multi-arch-enabling-or-disabling-multiarch-operator-using-labels.adoc

```diff
@@ -0,0 +1,35 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multiarch-tuning-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-enabling-or-disabling-multiarch-operator-using-labels_{context}"]
+
+= Enabling or disabling the Multiarch Tuning Operator by removing the labels in a namespace
+
+After creating a `PodPlacementConfig` custom resource (CR), you can manually enable or disable the Multiarch Tuning Operator in a namespace by removing the `multiarch.openshift.io/exclude-pod-placement` or `multiarch.openshift.io/include-pod-placement` labels.
```

Comment:
```suggestion
When creating a `PodPlacementConfig` custom resource (CR), you can setup a namespaceSelector to filter-in or filter-out the namespaces where the operand should process the node affinity of the created pods.

The namespaceSelector decides whether to run the webhook on a request for a namespaced resource (or a Namespace object), based on whether the namespace's labels match the selector.
```

I would specify two ClusterPodPlacementConfig excerpts (mostly including the different namespaceSelector field) as two paragraphs in the next

Reply:
Updated as per the suggestion.